test/controllers/workshops_controller_test.rb

```diff
@@ -33,14 +33,19 @@ class WorkshopsControllerTest < ActionDispatch::IntegrationTest
     assert_select 'title', 'Personal Website | Hack Club Workshops'
   end
 
-  test 'passes metadata to analytics on workshop view' do
+  test 'passes correct page tracking metadata' do
```

Comment:
My understanding is this tests that the correct boolean value for the should_track_pageview metadata is being passed to the frontend.